src/helm/ralph/templates/secrets.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.ralph.secret_name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "ralph.labels" . | nindent 4 }}
+  annotations:
+    checksum/config: {{ (tpl (.Files.Glob "vault.yaml").AsSecrets . ) | sha256sum }}
+type: Opaque
+data:
+{{- $v := $.Files.Get "vault.yaml" | fromYaml }}
+{{- range $key, $val := $v }}
+    {{ if contains "RALPH_" $key }}
+        {{ $key | nindent 2 }}: {{ $val | b64enc }}
+    {{- end }}
+{{- end }}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.ralph_lrs.auth_secret_name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "ralph.labels" . | nindent 4 }}
+  annotations:
+    checksum/config: {{ (tpl (.Files.Glob "vault.yaml").AsSecrets . ) | sha256sum }}
+type: Opaque
+data:
+{{- $v := $.Files.Get "vault.yaml" | fromYaml }}
+{{- range $key, $val := $v }}
+    {{ if eq "LRS_AUTH" $key }}
+        {{ "auth.json:" | nindent 2 }}: {{ $val | toJson | b64enc }}
+    {{- end }}
+{{- end }}
+
+
+{{- if .Values.ralph.elastic.mount_ca_secret }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.ralph.elastic.ca_secret_name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "ralph.labels" . | nindent 4 }}
+  annotations:
+    checksum/config: {{ (tpl (.Files.Glob "vault.yaml").AsSecrets . ) | sha256sum }}
+type: Opaque
+data:
+{{- $v := $.Files.Get "vault.yaml" | fromYaml }}
+{{- range $key, $val := $v }}
+    {{ if eq "ES_CA_CERTIFICATE" $key }}
+        {{ "es-cluster.pem:" | nindent 2 }}: {{ $val | b64enc }}
+    {{- end }}
+{{- end }}
```

Comment:
ditto

Reply:
same answer as before, I'll resolve this comment.